app/models/user.rb

```diff
@@ -1,6 +1,6 @@
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable and :omniauthable
+  # :lockable, :timeoutable and :omniauthable
```

Comment:
What do you think about getting rid of unused commented code?

Reply:
I would leave in for now. Will see how I would implement omniauth,Then - yes, I will get rid of it